ivy/functional/frontends/numpy/statistics/order_statistics.py

```diff
@@ -104,3 +108,29 @@ def nanpercentile(
                     arrayofpercentiles.append(_cpercentile(ii, i))
                 resultarray.append(arrayofpercentiles)
         return resultarray
+
+
+
+@ivy.numpy_frontend
+@ivy.handle_numpy_out
+@ivy.handle_numpy_dtype
+@ivy.to_ivy_arrays_and_back
+@ivy.handle_numpy_casting
+@ivy.from_zero_dim_arrays_to_scalar
+def quantile(
+    a,
+    q,
+    /,
+    *,
+    axis=None,
+    out=None,
+    overwrite_input=False,
+    method="linear",
+    keepdims=False,
+    interpolation=None,
+):

```

Comment:
I would recommend copying the function signature completely as I previously mentioned from [numpy's documentation](https://numpy.org/doc/stable/reference/generated/numpy.quantile.html) or [numpy's github](https://github.com/numpy/numpy/blob/v1.25.0/numpy/lib/function_base.py#L4292-L4544). This is because not all the arguments are forced to be keyword arguments like how you've written it here